installer/src/main/bin/encryptModel.cmd

```diff
@@ -79,11 +79,14 @@ ECHO         oracle_home     - the existing Oracle Home directory for the domain
 ECHO                           This is required unless the ORACLE_HOME environment
 ECHO                           variable is set.
 ECHO.
-ECHO         model_file      - the location of the model file to use.
+ECHO         model_file      - the location and name of the model file to use, or
+ECHO                           the locations and names of each model in a set of
+ECHO                           models. All models will be written back to the same
+ECHO                           location and name.
```

Comment:
Maybe clarify that comma-separated list for multiples? other tools say "the location of the model file to use.  This can also be specified as a  comma-separated list of model locations, ..."